romancal/regtest/test_elp_tvac.py

```diff
@@ -0,0 +1,166 @@
+"""Test that TVAC/FPS run in the pipeline"""
+
+import pytest
+import roman_datamodels as rdm
+from gwcs.wcstools import grid_from_bounding_box
+
+from romancal.pipeline.exposure_pipeline import ExposurePipeline
+
+from .regtestdata import compare_asdf
+
+# mark all tests in this module
+pytestmark = pytest.mark.bigdata
+
+
+@pytest.fixture(scope="module")
+def run_elp(rtdata_module):
+    rtdata = rtdata_module
+
+    # Get reference
+    rtdata.get_data("references/dark_ma510.asdf")
+
+    input_data = "TVAC2_NOMOPS_WFIFLA_20240419194120_WFI01_uncal.asdf"
+    rtdata.get_data(f"WFI/image/{input_data}")
+    rtdata.input = input_data
+
+    # Test Pipeline
+    output = "TVAC2_NOMOPS_WFIFLA_20240419194120_WFI01_cal.asdf"
+    rtdata.output = output
+    args = [
+        "roman_elp",
+        rtdata.input,
+        "--steps.dark_current.override_dark=dark_ma510.asdf",
+        "--steps.rampfit.override_dark=dark_ma510.asdf",
+    ]
+    ExposurePipeline.from_cmdline(args)
+
+    # get truth file
+    rtdata.get_truth(f"truth/WFI/image/{output}")
+    return rtdata
+
+
+@pytest.fixture(scope="module")
+def output_filename(run_elp):
+    return run_elp.output
+
+
+@pytest.fixture(scope="module")
+def output_model(output_filename):
+    with rdm.open(output_filename) as model:
+        yield model
+
+
+@pytest.fixture(scope="module")
+def truth_filename(run_elp):
+    return run_elp.truth
+
+
+@pytest.mark.soctests
+def test_output_matches_truth(output_filename, truth_filename, ignore_asdf_paths):
+    diff = compare_asdf(output_filename, truth_filename, **ignore_asdf_paths)
+    assert diff.identical, diff.report()
+
+
+@pytest.mark.soctests
```

Comment:
This looks like mostly a copy of [test_wfi_image_pipeline.py](https://github.com/spacetelescope/romancal/blob/826a972ebe8d01a6727921bfd248f029cb62255c/romancal/regtest/test_wfi_image_pipeline.py#L207). If the tests beyond this point in the file are not needed to satisfy the commented DMS requirements I'd say removing everything in this file after this commented line (64) makes sense.

If there is reason to keep the additional tests in this file at least `test_elp_input_dm` should be removed as it doesn't process tvac data and looks to be an exact duplicate of the one in `test_wfi_image_pipeline.py`.

Reply:
Though looking now, I would suggest keeping the tests but removing the dms requirement markings. Reason is that tvac/fps are taken in a different way, and having all the test to verify what is produced I actually like. Thoughts?

Reply:
as noted below, my cutting missed it, even after calling it out specifically 😞